.github/workflows/bump_dynamic_dependencies.yml

```diff
@@ -0,0 +1,43 @@
+# This workflow will generate a distribution and upload it to PyPI
+
+name: Bump Dynamic Dependencies
+on:
+  workflow_call:
+
+jobs:
+  build_and_publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.ref_name }}
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+      - name: Install Setuptools
+        run: |
+          python -m pip install -U pip setuptools
+      - name: Install Build Tools
+        run: |
+          python -m pip install build wheel
+      - name: Increment Version
+        run: |
+          VER=$(python setup.py --version)
+          python version_bump.py -b ${{ github.ref_name }}
```

Comment:
This should probably be removed. This increments the version, but it appears there will not be any releases associated with that version tag (Container or GitHub)